code/game/turfs/simulated/floor.dm

```diff
@@ -165,6 +165,8 @@ var/list/icons_to_ignore_at_floor_init = list("damaged1","damaged2","damaged3","
 
 /turf/simulated/floor/narsie_act()
 	if(prob(20))
+		icon_state = "cult"
+		icon_regular_floor = "cult"

```

Comment:
this would make cult floor tile always look like cult tile even if you replaced it with a normal floor tile.